VERSION

```diff
@@ -1,4 +1,4 @@
-VERSION="2.10.1"
+VERSION="2.10.2"
```

Comment:
reminder to rebase: this should be the last commit of the PR (where we will make the release)